src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -0,0 +1,28 @@
+package edu.oregonstate.mist.locations.mapper
+
+import edu.oregonstate.mist.locations.core.Location
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+
+class LocationMapper implements ResultSetMapper<Location> {
+    public Location map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        new Location(
+            id: rs.getInt("id"),
```

Comment:
Thanks for the comment. I looked at both examples when I was working on this API :) . I will leave this piece of code as is (based on this explanation). Actually the sample code that you link is not very efficient. It first creates a variable, calls several setter methods and then returns that variable after it.

I wanted to use a different approach. This example, takes advantage of groovy in a couple of ways:
* there's no return type since it's not needed. The last statement is automatically returned by the method. There's no need to assign the object to a variable and then return it right away.
* This is a single all to the map constructor that groovy adds to Plain Old Groovy Objects (POGOs). Instead of calling the setter method multiple times within the "with" block, all the parameters are passed to the constructor.

I personally don't like this piece of coding. Having to manually specify what each column gets mapped to specially when that doesn't change for the Location.groovy seems not very elegant. With Hibernate you can use annotations in the POGO class (Location.groovy) to specify what sql column a groovy property maps to. That is a bit more elegant than having to do it this way.